### ISSUE TITLE
Error in Kong v3.3

### ISSUE THREAD

**Issue by asofyana** [Optum/kong-splunk-log#18]
This plugin doesn't work in Kong v 3.3, because the module kong.tools.batch_queue has been renamed to kong.tools.queue (https://docs.konghq.com/gateway/changelog/)
After change kong.tools.batch_queue to kong.tools.queue, we have another error
2023/07/25 13:05:51 [error] 2386#0: *61025 failed to run log_by_lua*: /opt/kong/plugins/kong-splunk-log/handler.lua:167: attempt to call field 'new' (a nil value)
stack traceback:
        /opt/kong/plugins/kong-splunk-log/handler.lua:167: in function </opt/kong/plugins/kong-splunk-log/handler.lua:143>
        /usr/local/share/lua/5.1/kong/init.lua:433: in function 'execute_collected_plugins_iterator'
        /usr/local/share/lua/5.1/kong/init.lua:1673: in function 'log'

**Comment by jeremyjpj0916** [comment]
yep we were working on a 3.x compatible version, still needs testing though more.
